script.js

```diff
@@ -0,0 +1,197 @@
+    if (bookId === id) {
+      const { target } = e;
+      if (read === 'Readed') {
+        obj.isReaded = 'Pending';
+        target.innerText = 'Pending';
+      } else {
+        obj.isReaded = 'Readed';
+        target.innerText = 'Readed';
+      }
+    }
+    localStorage.setItem('library', JSON.stringify(myLibrary));
+  });
+}
+
+function deleteRow(e) {
+  let element;
+  const { target: { nodeName } } = e;
+  if (nodeName === 'svg') {
+    const { target: { parentNode: { id } } } = e;
+    element = id;
+  } else if (nodeName === 'path') {
+    const { target: { parentNode: { parentNode: { id } } } } = e;
+    element = id;
+  } else {
+    const { target: { id } } = e;
+    element = id;
+  }
+  element = element.split('-');
+  element = Number(element.pop());
+
+  let index = 0;
+  const newLib = [];
+  myLibrary.forEach(obj => {
+    const { id: bookId } = obj;
+    obj.id = index + 1;
+    if (bookId !== element) {
+      newLib.push(obj);
+      index += 1;
+    }
+  });
+  localStorage.setItem('library', JSON.stringify(newLib));
+  location.reload(); // eslint-disable-line no-restricted-globals
+}
+
+function maxWidth(e) {
+  const { target: { textLength } } = e;
+  if (textLength > 4) {
+    e.target.value = '';
+  }
+}
+
+/** Render all Array Elements */
+function render() {
+  let counter = 1;
+  myLibrary.forEach(obj => {
+    const mainContainer = document.createElement('div');
+    mainContainer.setAttribute('name', `book-${counter}`);
+    mainContainer.setAttribute('class', 'book');
+    Object.values(obj).forEach(value => {
+      if (value === 'Readed' || value === 'Pending') {
+        const btn = document.createElement('button');
+        btn.innerText = value;
+        btn.id = `btn-${mainContainer.getAttribute('name')}`;
+        btn.onclick = changeStatus;
+        const span = document.createElement('span');
+        span.appendChild(btn);
+        mainContainer.appendChild(span);
+      } else {
+        const spanValue = document.createElement('span');
+        spanValue.innerText = value;
+        mainContainer.appendChild(spanValue);
+      }
+    });
+    const delteDiv = document.createElement('div');
+    delteDiv.setAttribute('class', 'delete-div');
+    const delteBtn = document.createElement('a');
+    delteBtn.innerHTML = `<i id="row-${counter}" class="btn fas fa-backspace"></i>`;
+    delteBtn.setAttribute('class', 'delete-btn');
+    delteBtn.onclick = deleteRow;
+    delteDiv.appendChild(delteBtn);
+    mainContainer.appendChild(delteDiv);
+    counter += 1;
+    document.getElementById('display').appendChild(mainContainer);
+  });
+}
+
+function addBookToLibrary() {
+  const inputs = document.getElementById('bookSubmit');
+  const book = new Book();
+  Object.keys(book).forEach(key => {
+    if (key !== 'isReaded' && key !== 'id') {
+      book[key] = inputs[key].value;
+    } else if (key === 'id') {
+      book[key] = myLibrary.length + 1;
+    }
+  });
+  myLibrary.push(book);
```

Comment:
Apply minimum security of checking whether user has filled form before submission........
Else we get something like the screenshot below

![Screen Shot 2020-04-11 at 5 28 04 PM](https://user-images.githubusercontent.com/40712459/79049150-9e4b8f00-7c19-11ea-8daf-783c2ea2adf2.png)